lib/nats.js

```diff
@@ -1382,6 +1382,7 @@ Client.prototype.processErr = function(s) {
         this.closeStream();
     } else if (m.indexOf(PERMISSIONS_ERR) !== -1) {
         this.emit('permission_error', new NatsError(s, NATS_PROTOCOL_ERR));
+        this.inbound = null;
```

Comment:
Why not calling ` this.closeStream();`?

Reply:
@StarpTech I don't think it's a terminal error. In our case this was happening on `publish()` calls to `subjects` that the user wasn't allowed to publish to. Other/allowed publish/subscribe activity could proceed as before. The `Golang` library does not treat this as a terminal error either.

Reply:
Thanks for the insight.